ucm2/Toradex/apalis-imx6/HiFi.conf

```diff
@@ -0,0 +1,13 @@
+# Use case configuration for Toradex Apalis iMX6
```

Comment:
Use ucm2/NXP paths (like current NXP/iMX8 etc.). We prefer to store configurations for similar hardware in one place.

Reply:
@perexg : can you explain better what would you like? These UCM files are related to Toradex board, the actual sound card on these boards does not have much in common with NXP nor i.MX8 boards..

Or maybe you want the boards that use the `fsl-asoc-card` to be grouped together? How?